src/c_commands.cpp

```diff
@@ -67,7 +65,7 @@ static void CmdSet(CommandArguments args)
 
 static void CmdQuit(CommandArguments args)
 {
-    exit(0);
+    I_Quit();
```

Comment:
Does this compile with your compiler?  I'm getting a compiler error complaining that the function is missing.  You probably need to include i_system.h.